src/AccessibilityInsights.SharedUx/Properties/Resources.resx

```diff
@@ -898,6 +898,58 @@ Accessibility Insights Support Team
   <data name="btnFileBugAutomationPropertiesName1" xml:space="preserve">
     <value>File a bug</value>
   </data>
+  <data name="StartupModeControl_moreInformationText" xml:space="preserve">
```

Comment:
Should these resources be removed from the string resources in the AccessibilityInsights project?

Reply:
Yes

Reply:
Removed